CMakeLists.txt

```diff
@@ -172,6 +172,9 @@ set(Launcher_DISCORD_URL "https://prismlauncher.org/discord" CACHE STRING "URL f
 # Subreddit URL
 set(Launcher_SUBREDDIT_URL "https://prismlauncher.org/reddit" CACHE STRING "URL for the subreddit.")
 
+# Linux bwrap args
+set(Launcher_LINUX_BWRAP_EXTRA_ARGS "" CACHE BOOL "Extra args for bwrap sandboxing")
```

Comment:
Could there also be a runtime config, like via envvar? Users may want to pass extra paths to the sandbox without rebuilding the launcher.

Reply:
I am not sure if we want env vars, but I was definitely planning to add a UI setting for extra args.